src/Routes.js

```diff
@@ -21,6 +22,8 @@ function checkPaths(technology, app) {
 }
 
 export const Routes = () => {
+    const { getRegistry } = useContext(RegistryContext);
+    getRegistry().register(reducers);
```

Comment:
Couldn't this possibly register reducers on each render?

Reply:
Good point, I've wrapped it in `useEffect` that checks for `getRegistry` as it is mutated once app fully renders.